plasma-contract.Dockerfile

```diff
@@ -12,5 +12,5 @@ RUN apk add --update \
 		git
 
 RUN git clone https://github.com/omisego/plasma-contracts.git
-RUN cd /home/node/plasma-contracts && git checkout 3c5487c
+RUN cd /home/node/plasma-contracts && git checkout 5ce7d0b
```

Comment:
Nice bump, did this affect the conformance tests? Should we add additional conformance tests for the signatures?

Reply:
Conformance tests couldn't run without this bump. I'll check if we need to add more for this PR.